sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -62,6 +62,8 @@ end-to-end or SYCL-CTS tests.
 - Add a helpful comment describing what the test does at the beginning and
   other comments throughout the test as necessary.
 
+- Start all variables' name in llvm/sycl `.h` and `.cpp` (weather template or function variables) with underscore (`_`).
```

Comment:
I don't think it's a good idea.
In my opinion, [current guideline](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly) is much better option.

Reply:
How could we deal with variables with short length (length <= 2) in that case?

Reply:
- for variables of length = 2, there is no problem
- for 1-char variables, provide more descriptive names.

Reply:
Will it be better if I change rule from `starting with `_`` to `minimal length of variable = 2` ?

Reply:
I think for the problem you are trying to solve the solution would be to require at least one lower case letter. Am I right?

Reply:
Yes, you are. For example, use `Num` instead of `N` if we choose a `len(variable name) >=2` or `_N` instead of `N` in case of using underscores.

Reply:
As for me, both options are good

Reply:
`Num` is more descriptive and already aligned with LLVM Coding Standards, so it should be a preferred option.

Reply:
Ok, so I change that rule.